src/Compilers/CSharp/Portable/Binder/Semantics/Operators/OperatorKind.cs

```diff
@@ -313,6 +317,7 @@ internal enum BinaryOperatorKind
         And = 0x00001D00,
         Xor = 0x00001E00,
         Or = 0x00001F00,
+        Range = 0x00002000,
```

Comment:
>Range = 0x00002000, [](start = 8, length = 19)

This feels unexpected. Range expression is not "true" binary operator

Reply:
removed that handling now.

---
In reply to: [185144315](https://github.com/dotnet/roslyn/pull/26405#discussion_r185144315) [](ancestors = 185144315)